src/TrustedShops.php

```diff
@@ -82,6 +82,27 @@ public static function woocommerce_thankyou_order_received_text($text, $order) {
   <span id="tsCheckoutOrderAmount">{$order->get_total()}</span>
   <span id="tsCheckoutOrderCurrency">{$order->get_currency()}</span>
   <span id="tsCheckoutOrderPaymentType">{$order->get_payment_method()}</span>
+EOD;
+    foreach ($order->get_items() as $item) {
+      $product = $item->get_product();
+      $product_id = $product->get_id();
+      $product_url = get_permalink($product_id);
+      $product_image = wp_get_attachment_url($product->get_data()['image_id']);
+      $product_sku = $product->get_sku();
+      $product_gtin = get_post_meta($product_id, '_custom_gtin');
```

Comment:
Pass the third argument `TRUE` here to get a single value in order to avoid the `[0]` in the output below.